Js add specs and cli docs (#3771)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/metrics-overview.md

* `constraint`: For any type of metric, you may optionally include a constraint string, which applies a dimensional filter when computing the metric. You may think of this as your WHERE clause.  
* `meta`: Additional metadata you want to add to your metric. 

> An example of the complete metrics spec configuration is below:

Reviewer: change to active voice

```suggestion
Here's a complete example of the metrics spec configuration:

```
